tests/ping.go

```diff
@@ -98,3 +99,36 @@ func vmiConsoleExpectBatch(vmi *v1.VirtualMachineInstance, expected []expect.Bat
 	}
 	return err
 }
+
+// PingAppJob performs a netcat check (tcp by default) using a pod.
+// Optional port and arguments for the netcat command may be provided.
+// Returns the job deletion/cleanup function and an error.
+// The caller is expected to use the returned function to cleanup the resources.
+func PingAppJob(host, port string, args ...string) (func() error, error) {
```

Comment:
I think having 'ping' in the name is misleading - I immediately think ICMP ping. This is a simple TCP client afaiu. Why not name it for what it is ?

Reply:
The ping verb [definition](https://www.dictionary.com/browse/ping) and my understanding of it talks about a connectivity check and adding "App" or "Application" should make it clear what  it does.. especially when we add a doctext and port to the arguments.

I managed to convinced this another reviewer which had the same comment as you do. Hope you'll get convinced as well.
If you do have another catchy name without starting to enter implementation details please suggest it. This was the closest I could find/think.